packages/design-system-tailwind-preset/package.json

```diff
@@ -51,13 +51,14 @@
     "@types/jest": "^27.4.1",
     "deepmerge": "^4.2.2",
     "jest": "^27.5.1",
-    "tailwindcss": "^3.4.13",
+    "postcss": "^8.4.47",
     "ts-jest": "^27.1.4",
     "typedoc": "^0.24.8",
     "typedoc-plugin-missing-exports": "^2.0.0",
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
+    "@metamask/design-tokens": "^4.0.0",
```

Comment:
Specifying @metamask/design-tokens as a peer dependency with a minor version lock (e.g., "^4.0.0"). This allows for patch updates while ensuring compatibility with new tokens we might rely on in the preset.